docs/src/main/paradox/events-by-tag.md

```diff
@@ -137,6 +137,15 @@ be taken not to have batches that will be rejected by Cassandra. Two other cases
 * Periodically: By default 250ms. To prevent eventsByTag queries being too out of date.
 * When a starting a new timebucket, which translates to a new partition in Cassandra, the events for the old timebucket are written.
 
+## Cleanup of tag_views table
+
+By default the tag_views table keeps tagged events indefinitely, even when the original events have been removed. 
+Depending on the volume of events this may not be suitable for production.
+
+Before going live decide a time to live (TTL) and, if small enough, consider using the [Time Window Compaction Strategy](http://thelastpickle.com/blog/2016/12/08/TWCS-part1.html).
+See `events-by-tag.time-to-live` in reference.conf for how to set this.
```

Comment:
"Note that if there is a delay between tagged events bigger than time-to-live [the office goes on fire]" or something else about the backside of setting this wrong?